### ISSUE TITLE
Versioned yaml manifests

### ISSUE THREAD

**Issue by NissesSenap** [projectcalico/calico#4729]
## Expected Behavior

I want to be able to specify a manifest version when installing calico on for example EKS to get consistent installations no matter when I install my cluster.

```
kubectl apply -f https://raw.githubusercontent.com/projectcalico/calico/v3.19.1/manifests/calico-vxlan.yaml
```

## Current Behavior

Today the docs https://docs.projectcalico.org/getting-started/kubernetes/managed-public-cloud/eks want's me to perfrom:

```shell
kubectl apply -f https://docs.projectcalico.org/manifests/calico-vxlan.yaml
```

But I don't know if you change the release between my install today and in a month or two.
Today this is not possible due to your helm templating solution.

## Possible Solution

To be backwards compatible I would create a new folder called generated_manifests or similar where you store the generated manifests and you check it in when you create a new tag.
This way you can still use the docs page that most people should use to get the latest and for us that want a manifest of tag X can do that using the github page.

## Context

Want to know that I get a **consistent** experience over all my clusters no matter when I install the cluster.
In short I want to be able to use the power of dev/qa/prod

## Your Environment

N/A

**Comment by NissesSenap** [comment]
There is a archive feature on the homepage:
You can for example do something like: https://docs.projectcalico.org/archive/v3.19/manifests/calico-vxlan.yaml
